.travis.yml

```diff
@@ -18,7 +18,7 @@ script:
   - if [[ "$(go version)" != *go1.8* ]]; then go vet ./...; fi
   - lint_warns=$(golint ./... | grep -v vendor/) || true
   - if [ ! -z "$lint_warns" ]; then echo "$lint_warns"; exit 1; fi
-  - go test -cpu=1,2 -coverprofile=coverage.txt -covermode=count gopkg.in/src-d/go-license-detector.v1
+  - go test -cpu=1,2 gopkg.in/src-d/go-license-detector.v1/...
```

Comment:
Are you sure this does not break the coverage?

Reply:
I'll run `go test` on everything but do the coverage in a single package (support for many packages comes with go1.10)

Reply:
:+1: